llvm/lib/Target/AMDGPU/GCNHazardRecognizer.h

```diff
@@ -13,6 +13,7 @@
 #ifndef LLVM_LIB_TARGET_AMDGPUHAZARDRECOGNIZERS_H
 #define LLVM_LIB_TARGET_AMDGPUHAZARDRECOGNIZERS_H
 
+#include "SIInstrInfo.h"
```

Comment:
Don't see why you would need to move the include here

Reply:
I need `SIInstrInfo::getNumWaitStates` as a default value in `getWaitStatesSince` declaration.